server/enterprise/message_export/shared/export_data.go

```diff
@@ -36,11 +36,12 @@ type PostExport struct {
 	UpdateAt    int64 // if this is an updated post, this is the updated time (same as deleted time for deleted posts).
 
 	// when a message is edited, the EditedOriginalMsg points to the message Id that now has the newly edited message.
-	EditedNewMsgId string
-	Message        string            // the text body of the post
-	PreviewsPost   string            // the post id of the post that is previewed by the permalink preview feature
-	Attachments    []*model.FileInfo // the post's attachments
-	FileInfo       *model.FileInfo   // if this was a file PostExport, FileInfo will contain that info. Otherwise, nil.
+	EditedNewMsgId    string
+	Message           string                   // the text body of the post
+	PreviewsPost      string                   // the post id of the post that is previewed by the permalink preview feature
+	AttachmentStarts  []*FileUploadStartExport // the post's attachments that were uploaded this export period
```

Comment:
`Starts` is a weird suffix here, because then I would expect `Ends` to also be there. I think `AttachmentCreates/Deletes` may be a better alternative. 0/5